packages/paste-website/package.json

```diff
@@ -53,6 +53,11 @@
     "@types/lodash": "^4.14.136",
     "@types/react-helmet": "^5.0.9",
     "babel-preset-gatsby": "^0.2.8",
+    "gatsby-image": "^2.2.8",
```

Comment:
My only quesiton is that do these need to be deps and not dev deps? Does Gatsby need them outside of the build step?

Reply:
You're right. Those should be deps. I mis-translated the `npm` command to `yarn`. I'll update those.